inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -175,15 +252,22 @@ int main(int argc, char* argv[]) {
             load_config(FLAGS_load_config, config);
         }
 #endif
-        /** This vector stores paths to the processed images **/
-        std::vector<std::string> inputFiles;
-        parseInputFilesArguments(inputFiles);
+        /** This vector stores paths to the processed images with input names**/
+        auto inputFiles = parseInputArguments();
 
         // ----------------- 2. Loading the Inference Engine
         // -----------------------------------------------------------
         next_step();
 
         Core ie;
+        if (FLAGS_d == "TEMPLATE") {
+#ifdef _DEBUG
+            ie.RegisterPlugin("templatePlugind", "TEMPLATE");
+#else
+            ie.RegisterPlugin("templatePlugin", "TEMPLATE");
```

Comment:
I think we don't need this code. At least I see:
```
[ ERROR ] Device with "TEMPLATE"  is already registered in the InferenceEngine
```

Reply:
Where this message is occurred?  Because when I removed this code, I got 

`[ ERROR ] Device with "TEMPLATE" name is not registered in the InferenceEngine`

Reply:
@ivikhrev on my local build from your branch

Reply:
@nkogteva  hm, maybe I need to rebuild my solution.

Reply:
@nkogteva is it WIndows or Linux you testing on?

Reply:
@ivikhrev Linux

Reply:
@nkogteva I've tested on both platforms, and it seems is is already registered only on Linux, on Windows the error will arise if we remove this code.